src/overmap_ui.cpp

```diff
@@ -1158,8 +1158,7 @@ static void draw_om_sidebar(
         }
     }
 
-    mvwprintz( wbar, point( 1, 12 ), input_context::get_hint_color(),
-               _( "Use movement keys to pan." ) );
+    print_colored_text( wbar, point( 1, 12 ), inp_ctxt->get_hint_directions( _( "Pan view" ) ) );
```

Comment:
Those are keys I'm using for moving around in cardinal directions (I've got a weird keyboard layout).

With default settings it shows the following:
![image](https://user-images.githubusercontent.com/2420411/228556277-4d86cc02-54cd-4327-97ae-4db0f4ea5e56.png)

Which seems reasonable although I suspect that most players don't play with vi-keys and use the keypad instead (or maybe even arrow keys and avoid moving diagonally). It's hard to know what players are actually using to move around and thus hard to know what we should display here.

I'm starting to think that because movement like this is the default "action" players are doing, we can avoid displaying any sort of keybinds related to this because if the player has gotten this far, they know how to move around their view/character (however they like to do so) and we probably don't need to remind them.

What do you think?

Reply:
That's actually a good point.

Reply:
I don't have a strong preference for either so I'll leave that up to you.